src/utils/ComponentsTypes.ts

```diff
@@ -0,0 +1,22 @@
+import { Dom, $$ } from './Dom';
+import { get } from '../ui/Base/RegisteredNamedMethods';
+import { Component } from '../ui/Base/Component';
+
+export class ComponentsTypes {
+  public static get allFacetsType() {
+    return ['Facet', 'FacetSlider', 'FacetRange', 'TimespanFacet', 'HierarchicalFacet', 'CategoryFacet'];
+  }
+
+  public static get allFacetsClassname() {
+    return ComponentsTypes.allFacetsType.map(type => `Coveo${type}`);
```

Comment:
Would it be better to use `Component.computeCssClassForType` instead of hard-cording `Coveo` prefix?